cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -536,6 +554,10 @@ class FunctionScheduleLoader(
     list_write_cls = FunctionScheduleWriteList
     dependencies = frozenset({FunctionLoader})
 
+    def __init__(self, client: CogniteClient):
+        super().__init__(client)
+        self.extra_configs: dict[str, Any] = {}
```

Comment:
If this a generic concept we now introduce, I think it should be set on the base class.